packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -54,10 +54,22 @@ export const setTourType = (param: string) => {
     tour_type.key = param;
 };
 
+export const tour_status_ended = {
```

Comment:
@farabi-deriv why we haven’t typed this object in the file .tsx?

Reply:
done ✅